_posts/2023-01-11-sigstore-in-toto-opa.md

```diff
@@ -0,0 +1,70 @@
+---
+title: "Sigstore, in-toto, OPA, orientation"
+date: 2023-01-11
+author: Daniel Bevenius
+---
+
+This post tries to explain what these project are and how they complement each
+other.
+
+As someone who was completly new to secure supply chain security (sscs) there
+were a lot of new projects that I learned the names of but did not really
+understand how they all fit together. This post hopes to clarify a few of these
+projects, and others will be addressed in future posts.
```

Comment:
Would this make a better opening paragraph (which will become a blurb on the index)? It pretty much repeats what your current first para/blurb says, but more personally, which I like.

I do like the "how they complement each other" phrase, though. Maybe merge both paras somehow?

Reply:
I'm made an attempt of merging this with the first section. Please take a look and see what you think? Thanks!

Reply:
Perfect!